Pod/Classes/PINAnimatedImageManager.m

```diff
@@ -41,15 +46,22 @@ + (instancetype)sharedManager;
 
 @implementation PINAnimatedImageManager
 
-+ (void)initialize
++ (void)load
```

Comment:
Yeah, so the intention was to clear out any images even if no new GIFs were loaded. I tried to make this as lightweight as possible by only registering for a notification.

Wonder if there's an alternative to load to force the code to run at some point after launch…